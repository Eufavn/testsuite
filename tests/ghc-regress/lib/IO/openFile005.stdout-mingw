two writes (should fail)
Right {loc=openFile005.out1,type=writable,binary=False,buffering=block (512)}
write and an append (should fail)
Right {loc=openFile005.out1,type=writable (append),binary=False,buffering=block (512)}
read/write and a write (should fail)
Right {loc=openFile005.out1,type=writable,binary=False,buffering=block (512)}
read and a read/write (should fail)
Right {loc=openFile005.out1,type=read-writable,binary=False,buffering=block (512)}
write and a read (should fail)
Right {loc=openFile005.out1,type=readable,binary=False,buffering=block (512)}
two writes, different files (silly, but should succeed)
two reads, should succeed
