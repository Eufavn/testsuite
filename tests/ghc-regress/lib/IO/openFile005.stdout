two writes (should fail)
Left resource busy
Action: openFile
Reason: file is locked
File: openFile005.out1
write and an append (should fail)
Left resource busy
Action: openFile
Reason: file is locked
File: openFile005.out1
read/write and a write (should fail)
Left resource busy
Action: openFile
Reason: file is locked
File: openFile005.out1
read and a read/write (should fail)
Left resource busy
Action: openFile
Reason: file is locked
File: openFile005.out1
write and a read (should fail)
Left resource busy
Action: openFile
Reason: file is locked
File: openFile005.out1
two writes, different files (silly, but should succeed)
two reads, should succeed
